core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml

```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
```

Comment:
Do we need this?

Reply:
I added it from what we have in the WMCO prowconfig. I don't see a usecase so will update it.

Reply:
I tried to adjust this but the bot is not happy due to org wide restrictions:
`the following orgs or repos enable the bugzilla plugin but do not forbid the bugzilla/invalid-bug label for merging: [repo: openshift/prometheus-community-windows_exporter`
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/38362/pull-ci-openshift-release-master-config/1653797247189520384/build-log.txt

Reply:
Why don't we disable the bugzilla plugin in `_pluginconfig.yaml`?